Ccount.c

```diff
@@ -0,0 +1,20 @@
+/*Mariposa Lee, checking for c in a user input string. Researched a lot for this one, but still couldn't quite get it to work...*/
+#include <stdio.h>
+int main () {
+	char string[200], c, C;
+	int i, time;
+	time = 0;
+c = 'c';
+C = 'C';
+printf("Enter some characters.\n");
+	scanf("%c", &string);
+
+for (i=0 ; string[i]!='\0'; i++){
```

Comment:
your for loop isn't iterating through the characters. It's only checking the first character, which is why it only counts the first c if it exists.